packages/hierarchies/api/presentation-hierarchies.api.md

```diff
@@ -159,7 +159,8 @@ interface ECSqlValueSelector {
 
 // @beta
 export function extractFilteringProps(rootLevelFilteringProps: HierarchyFilteringPath[], parentNode: Pick<NonGroupingHierarchyNode, "filtering"> | undefined): {
-    filteredNodePaths: HierarchyFilteringPath[];
+    filterPathsIdentifierPositions?: Array<[number, number]>;
+    nodeIdentifierPaths: HierarchyFilteringPath[];
```

Comment:
This is a breaking change that we can't make anymore

Reply:
The other changes in this file are also breaking

Reply:
Changed